tests/query-node/run-tests.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -e
+
+SCRIPT_PATH="$(dirname "${BASH_SOURCE[0]}")"
+cd $SCRIPT_PATH
+mkdir -p hydra-test/
+cd hydra-test
+
+npx @dzlzv/hydra-cli scaffold --wsProviderUrl=ws://joystream-node:9944/ --projectName=Test
+
+cp ../../../types/augment/all/defs.json ./mappings/typedefs.json
+export TYPES_JSON=../../mappings/typedefs.json
+
+# Joystream mappings and graphsql schema
+# cp ../../../query-node/mappings/* ./mappings/
+# cp ../../../query-node/schema.graphql ./
+
+npx @dzlzv/hydra-cli codegen
```

Comment:
Probably a more isolated and thus safer way would be to run `db:migrate` from a docker container, and in this case we don't need to run `codegen` here at all:

```
docker build . -f docker/Dockerfile.hydra -t hydra-indexer:latest
docker-compose up $1 -d db
docker run --env DB_HOST=db --env TYPEORM_HOST=db --env-file .env --network=hydra-test_default hydra-indexer:latest yarn db:migrate 
```
and then
```
docker-compose up $1 -d
```

Reply:
sounds like a better approach yes, will update

Reply:
FYI, I just published hydra-cli@0.0.16 where I added a few convenenience yarn targets. The whole process can now be a bit streamlined: 
```
yarn docker:db:up # runs db from docker-compose
yarn docker:build # builds an image with the schema, mappings and types.json, it's called hydra-kit
yarn docker:db:migrate # migrate the db
yarn docker:up # runs docker-compose up -d
```

There scripts above are more or less repeat what we discuss above, so it may or may not be convenient to use them